simulator/modules/fetch/fetch.cpp

```diff
@@ -67,15 +67,27 @@ Target Fetch<FuncInstr>::get_target( Cycle cycle)
                                                                 rp_flush_target_from_decode->read( cycle) : Target();
     const Target branch_target   = rp_target->is_ready( cycle) ? rp_target->read( cycle) : Target();
 
+    if (prefetch_enabled && prefetch_method == "wrong_path")
+    {
```

Comment:
Our code guideline is to use as less brackets as possible: https://github.com/MIPT-ILab/mipt-mips/wiki/Code-style-guidelines#pythonesque-statements